google-analytics-chart.html

```diff
@@ -38,12 +56,18 @@
     </google-chart>
     <content select="footer"></content>
   </template>
+</dom-module>
 
-  <script>
+<script>
 
-    (function() {
+  (function() {
+    "use strict";
```

Comment:
I think @ebidel incorrectly used double-quotes in his comment. This should be single-quotes to match the style guide. Also, to answer your question, I think `'use strict'` should be added to all elements. Eric can correct me if I'm wrong.